Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -418,6 +422,14 @@ static Boolean isAXAuthenticatedCallback(audit_token_t auditToken)
 
     if (!isParentProcessAFullWebBrowser(*this))
         disableURLSchemeCheckInDataDetectors();
+
+    // Soft link frameworks related to Data Detection eagerly, since these are normally soft linked in every WebContent process.
```

Comment:
I think we want to say something like "Soft link frameworks related to Data Detection before we disconnect from launchd because these frameworks connect to launchd temporarily at link time to register XPC services. See <rdar:/...> (my feature request to stop doing that)".

Reply:
> I think we want to say something like "Soft link frameworks related to Data Detection before we disconnect from launchd because these frameworks connect to launchd temporarily at link time to register XPC services. See rdar:/... (my feature request to stop doing that)".

Thanks! I have updated the comment in the latest version of the patch.